news/2024/2024-07-23-osu-austria-conventions-2024.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: osu! Austria @ Conventions 2024
+date: 2024-07-23 12:00:00 +0000
+---
+
+The osu! Austria community proudly presents its appearance at not one, but two upcoming conventions in 2024!
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/Banner.png)
+
+Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. What are you waiting for? Get all the information you need down below!
+
+## AniNite 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
+
+Join us at the biggest anime and manga convention in Austria with over 25,000 visitors each year: **[AniNite](https://www.aninite.at/en/)**! Located near the beautiful Danube River at Austria Center in Vienna, you can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.
+
+During this time, we'll host a **battle royale–styled** osu! tournament. Make sure to register during the event and take home the trophy!
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at **[Yunicon 2024](https://www.yunicon.at/en/)**, situated at Mutliversum Schwechat. During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicupbannerfinals.png)
+
+You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
```

Comment:
In relation to [this](https://github.com/ppy/osu-wiki/pull/11802/files#r1688447290):
```suggestion
You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) where you can get notified when sign-ups open, and where organisation of the Qualifiers will happen.
```